panc/src/main/java/org/quattor/pan/parser/PanParserAstUtils.java

```diff
@@ -26,64 +26,8 @@ Centre National de la Recherche Scientifique (CNRS).
 import org.quattor.pan.dml.data.Element;
 import org.quattor.pan.dml.data.Null;
 import org.quattor.pan.dml.data.Undef;
-import org.quattor.pan.dml.functions.Append;
-import org.quattor.pan.dml.functions.Base64Decode;
-import org.quattor.pan.dml.functions.Base64Encode;
-import org.quattor.pan.dml.functions.Clone;
-import org.quattor.pan.dml.functions.Create;
-import org.quattor.pan.dml.functions.Debug;
-import org.quattor.pan.dml.functions.DebugSuppressed;
-import org.quattor.pan.dml.functions.Delete;
+import org.quattor.pan.dml.functions.*;
 import org.quattor.pan.dml.functions.Deprecated;
```

Comment:
Otherwise Java doesn't know whether it should use org.quattor.pan.dml.functions.Deprecated or java.lang.Deprecated